test/unit/lib/plugins/aws/package/compile/events/websockets/index.test.js

```diff
@@ -131,23 +133,43 @@ describe('test/unit/lib/plugins/aws/package/compile/events/websockets/index.test
             },
           },
         },
-      });
+      }));
     });
 
     it('should create a websocket api resource', () => {
+      const websocketsApiName = awsNaming.getWebsocketsApiName();
+      expect(cfTemplate.Resources.WebsocketsApi).to.deep.equal({
+        Type: 'AWS::ApiGatewayV2::Api',
+        Properties: {
+          Name: websocketsApiName,
+          RouteSelectionExpression: '$request.body.action',
+          Description: 'Serverless Websockets',
+          ProtocolType: 'WEBSOCKET',
+        },
+      });
       // Replaces
```

Comment:
Let's also remove this comment (they were just guide for refactor, with refactor they should be removed)

Reply:
@medikoo Comments are removed now with latest commit. Thank you for your help!